docs/installation.rst

```diff
@@ -190,19 +188,17 @@ This uses the latest (source package) release from PyPi.
                                # 0.41 and 0.41.1 have unicode issues at install time
     pip install borgbackup
 
-Note: we install into a virtual environment here, but this is not a requirement.
+.. note:: We install into a virtual environment here, but this is not a requirement.
 
 
 Installation (git)
 ------------------
 
 This uses latest, unreleased development code from git.
-While we try not to break master, there are no guarantees on anything.
-
-.. parsed-literal::
+While we try not to break master, there are no guarantees on anything. ::
 
-    # get |project_name| from github, install it
-    git clone |git_url|
+    # get borg from github
+    git clone https://github.com/borgbackup/borg.git
```

Comment:
that might be, but the correct fix then is to use the placeholders, if possible.